tasks/update-granules-metadata/tests/data/payload.json

```diff
@@ -0,0 +1,121 @@
+{
+    "config": {
+      "bucket": "cumulus-internal",
+      "buckets": {
+        "internal": {
+          "name": "cumulus-internal",
+          "type": "internal"
+        },
+        "private": {
+          "name": "cumulus-private",
+          "type": "private"
+        },
+        "protected": {
+          "name": "cumulus-protected",
+          "type": "protected"
+        },
+        "public": {
+          "name": "cumulus-public",
+          "type": "public"
+        }
+      },
+      "distribution_endpoint": "https://something.api.us-east-1.amazonaws.com/",
+      "collection": {
```

Comment:
I think you might not need this for your task, in which case it'd good to simplify the fixture